features/guidance/turn-angles.feature

```diff
@@ -0,0 +1,1104 @@
+                                                |
+                                                |
+                                                i
+            """
+
+        And the ways
+            | nodes | lanes | name  |
+            | abcd  | 3     | road  |
+            | defg  | 2     | road  |
+            | hdi   | 2     | cross |
+
+        When I route I should get
+            | waypoints | route           | turns                    |
+            | a,g       | road,road       | depart,arrive            |
+            | i,a       | cross,road,road | depart,turn right,arrive |
+            | i,g       | cross,road,road | depart,turn left,arrive  |
+            | h,g       | cross,road,road | depart,turn right,arrive |
+            | h,a       | cross,road,road | depart,turn left,arrive  |
+
+    #http://www.openstreetmap.org/#map=19/52.56243/13.32666
+    Scenario: Modelled Curve on Segregated Road
+        Given the node map
+            """
+              a                                       f
+              |                                   .   e - - - - g
+              |                           h  '        |
+              |                .    '                 |
+              |             i                         |
+              |         .                             |
+              |                                       |
+              |     .                                 |
+              |                                       |
+              |   j                                   |
+              |                                       |
+              | '                                     |
+              |.                                      |
+              |                                       |
+            m b - - - - - - - - - - - - - - - - - - - k - - - - l
+              |                                       |
+              c                                       d
+            """
+
+        And the ways
+            | nodes  | name | oneway | lanes |
+            | abc    | road | yes    | 3     |
+            | dkef   | road | yes    | 3     |
+            | mbkl   | turn | yes    | 3     |
+            | gehijb | turn | yes    | 3     |
+
+        When I route I should get
+            | waypoints | route          | turns                        |
+            | a,c       | road,road      | depart,arrive                |
+            | a,l       | road,turn,turn | depart,turn left,arrive      |
+            | a,f       | road,road,road | depart,continue uturn,arrive |
+            | d,f       | road,road      | depart,arrive                |
+            | d,l       | road,turn,turn | depart,turn right,arrive     |
+            | d,c       | road,road,road | depart,continue uturn,arrive |
+            | g,l       | turn,turn,turn | depart,continue uturn,arrive |
+            | g,c       | turn,road,road | depart,turn left,arrive      |
```

Comment:
If you remove the oneway, you sill have only very few meters in between. the two roads. We could possibly add some investigations if another turn, during collapsing, would also be a turn left at the new combined turn.

The turn angle computed at `e` turning from `g` to `h` is straight. Only the combined turn is given as left.